docs/java/java-project.md

```diff
@@ -79,6 +61,22 @@ If you have multiple subfolders that have source code for your project, you can
   <source src="/docs/java/java-project/multiple-source.mp4" type="video/mp4">
 </video>
 
+## Lightweight Mode
+
+VS Code for Java supports two modes, lightweight and standard. With lightweight mode, only source files and JDK are resolved by language server; with standard mode, imported dependencies are resolved and project is built by language server. Lightweight mode works best when you need a super quick-to-start and lightweight environment to work with your source files, for example, reading source code, navigating among source code and JDK, viewing outline and Javadoc, and detecting and fixing syntax errors. Also, code completion is supported within the scope of source files and JDK.
+
+Lightweight mode does neither resolve imported dependencies nor build project, so it does not support run, debugging, refactoring, linting or detecting semantic errors. For taking advantage of these features, you need switch your workspace from lightweight mode to standard mode.
+
+You can control which mode to launch with by configuring `java.server.launchMode` with below options:
+
+- `Hybrid` (default) - Firstly, a workspace is opened with lightweight mode. You will be asked whether switch to standard mode, when your source files and JDK are resolved by language server, or when an extension requires to run with standard mode. Or, you can switch by clicking server mode icon on status bar. Once a workspace is in standard mode, it cannot be switched back to lightweight mode.
+- `Standard` - A workspace is opened with standard mode. Standard mode cannot be switched to lightweight mode.
+- `LightWeight` - A workspace is opened with lightweight mode only. Cannot switch to standard mode unless changing this setting to Hybrid or Standard and reload your window.
+
+Status bar indicates which mode current workspace is in using different icons. `rocket`: workspace opened with lightweight mode; `synchronisation`: workspace being opened with standard mode; `thumb up`: workspace opened with standard mode. When in hybrid mode, clicking lightweight mode icon takes you to standard mode.
```

Comment:
Does vscode-docs support emoji?

rocket -> 🚀
sync -> 🔄
thumb up -> 👍

Reply:
@gregvanl, do you know?

Reply:
@yaojinyang Yes, here's an example https://vscode-docs-master.azurewebsites.net/updates/v1_47#_stable-windows-arm-builds. You don't want to rely completely on an emoji, since it isn't accessible but you can include an emoji along with the text description.

Reply:
Added screenshots for icons